test/test_mem_growth.c

```diff
@@ -5,7 +5,11 @@
 #include <emscripten/html5.h>
 #include <emscripten/heap.h>
 
+#ifdef __wasm64__
+#define EXPECTED_FINAL_HEAP_SIZE (4*1024*1024*1024ll)
```

Comment:
Interesting, why is wasm64 capable of allocating 64K more here?

Reply:
According to library.js:

```
    // Stay one Wasm page short of 4GB: while e.g. Chrome is able to allocate       
    // full 4GB Wasm memories, the size will wrap back to 0 bytes in Wasm side   
    // for any code that deals with heap sizes, which would require special         
    // casing all heap size related code to treat 0 specially.                   
    {{{ Math.min(MAXIMUM_MEMORY, FOUR_GB - WASM_PAGE_SIZE) }}},   
```

Reply:
I see, thanks. Maybe refer to that from here in the test?

Reply:
Done